rescheduler/rescheduler.go

```diff
@@ -112,6 +123,11 @@ func main() {
 				if len(criticalPods) > 0 {
 					for _, pod := range criticalPods {
 						glog.Infof("Critical pod %s is unschedulable. Trying to find a spot for it.", podId(pod))
+						k8sApp := "unknown"
+						if l, found := pod.ObjectMeta.Labels["k8s-app"]; found {
```

Comment:
Oh don't get me wrong, I'm all for doing this, I've been trying to push for this convention in [some](https://github.com/kubernetes/kops/issues/1226) bootstrapping tools. What I'm saying is, that it would be neat if we can formalize this to get consistency between clusters even if created by different tools. But that should rather be tracked somewhere else, I was just wondering if something like that already exists.